### DISCUSSION TITLE
Using CSS-selectors | How CSS-styling works in ha-floorplan and in general

### DISCUSSION THREAD

**Discussion by exetico** [ExperienceLovelace/ha-floorplan#258] [Videos]
We'll take a look at how CSS-selectors works, by using a Class and Id selector. I'll also try and give you a better understanding in, how the CSS style "connects" to the HTML-dom, and how ha-floorplan are updating the dom-element. Learn how to see the CSS-properties in your browser, and debug by toggling a given property on and off.

This is a (shorter) video all about CSS-styling in ha-floorplan, before I'll release part 2 of "Create your first floorplan in Inkscape".

❤️☕ Support my work:
https://www.buymeacoffee.com/coffeetoexetico

[![3 - ha-floorplan - How CSS-styling works in ha-floorplan and in general Thumbnail](https://user-images.githubusercontent.com/3549445/197844775-1a84cbc5-0bef-4105-a796-b8e0b9aa41c4.png)](https://youtu.be/oqO4Vx6U1Mo)

🗣️💬 Your feedback
Please let me know what you think, and press the like button! I'll also ask you to search for code-related inputs in our discussion area, instead of here on YouTube. But short questions are totally welcome, but I can't promise that I'll walk through all questions from now on.

Software used:
➡️ Home Assistant 2022.10.04
➡️ Firefox Developer Edition 105.0b4 (64-bit)

**Comment by OzGav** [comment]
Yeah thanks for this one Tobias. I learnt heaps about debugging CSS. It will help me help others as well!
